Makefile.Common

```diff
@@ -3,7 +3,7 @@ ALL_PKGS := $(sort $(shell go list ./...))
 # COVER_PKGS is the list of packages to include in the coverage
 COVER_PKGS := $(shell go list ./... | tr "\n" ",")
 
-GOTEST_OPT?= -race -timeout 120s
+GOTEST_OPT?= -race -timeout 120s -v
```

Comment:
I worry enabling verbose might trip up the amount of storage on the node?

Reply:
It's been fine. And given that the tests were misbehaving, it was needed to see what was going on. It could be in its own PR though.